src/components/ordval/guessOption.tsx

```diff
@@ -92,16 +91,38 @@ export const OrdvalGuessOption = ({
       opacity={showSolution ? (solution?.id === guess.id ? 1 : 0.5) : undefined}
       position="relative"
       w="full"
-      h={{ base: "16", md: "32", lg: "40" }}
       {...props}
     >
+      <Image
+        src={guess.pImage ?? guess.image}
+        alt={guess.desc}
+        objectFit="contain"
+        w="full"
+        h={showSolution ? { base: "25vh", md: "72", lg: "96" } : 0}
+        loading="lazy"
+        pointerEvents="none"
+        blendMode="darken"
+        visibility={showSolution ? 'visible' : 'hidden'}
+      />
+      <Box
+        display={showSolution ? 'none' : 'block'}
+        backgroundColor="gray.50"
+        filter="auto"
+        blur="10px" w="80%"
+        m="0 auto"
+        h={{ base: "28vh", md: "80", lg: "28rem" }}
```

Comment:
This differs from the image size, so there's a bit of difference between the guess and solution mode

Reply:
I improved them in the current version by making some adjustments. But this will be no more required if I position text on the bottom.